lib/matplotlib/backends/qt_editor/figureoptions.py

```diff
@@ -137,39 +137,43 @@ def prepare_data(d, init):
     # Is there a curve displayed?
     has_curve = bool(curves)
 
-    # Get / Images
-    imagedict = {}
-    for image in axes.get_images():
-        label = image.get_label()
-        if label == '_nolegend_':
+    # Get ScalarMappables.
+    mappabledict = {}
```

Comment:
Would probably do `mappable_*` instead of `mappable*` on compounds for better readability.

Reply:
There's also curvedict/curvedata/etc. for Line2D instances (just above), so if you want to get to change both of them...